### PR TITLE
Hey guys, after careful considering my updated code, please approve it.

### PR DESCRIPTION
**Pull request by paveldanek**

### CODE REVIEW

File: src/store/entities/Product.java [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
-	private static int idCounter = 1;
-
-	public Product(String name, double currentPrice, int stockOnHand, int reOrderLevel) {
+	public Product(String name, String id, double currentPrice, int stockOnHand, int reOrderLevel) {
 		this.name = name;
 		this.currentPrice = currentPrice;
 		this.stockOnHand = stockOnHand;
 		this.reOrderLevel = reOrderLevel;
-		this.id = "P-" + idCounter++;
```
Comment (carterclark) [inline src/store/entities/Product.java:18]: Could we keep this in? I thought it made sense to have member id's be M-# and product id's be P-#

Comment (carterclark) [inline src/store/entities/Product.java:18]: Could we keep this in? I thought it made sense to have member id's be M-# and product id's be P-#

**Review by carterclark — CHANGES_REQUESTED** [review]

File: src/store/entities/Product.java (hunk 1 above)
Comment (paveldanek) [reply src/store/entities/Product.java:18]: It does make sense but Brahma wants--in the case of Product--the product number be manually input...

Comment (carterclark) [inline src/store/entities/Product.java:18]: Would like this to be in

**Review by carterclark — CHANGES_REQUESTED** [review]

File: src/store/entities/Product.java (hunk 1 above)
Comment (carterclark) [reply src/store/entities/Product.java:18]: I see

**Review by carterclark — APPROVED** [review]

**Pull request merged**